pkg/apis/go.mod

```diff
@@ -36,24 +36,24 @@ require (
 	github.com/rancher/gke-operator v1.9.3
 	github.com/rancher/norman v0.0.0-20240708202514-a0127673d1b9
 	github.com/rancher/rke v1.6.3
-	github.com/rancher/wrangler/v3 v3.0.0
+	github.com/rancher/wrangler/v3 v3.0.1-rc.2
 	github.com/sirupsen/logrus v1.9.3
-	k8s.io/api v0.30.2
-	k8s.io/apimachinery v0.30.2
+	k8s.io/api v0.31.1
+	k8s.io/apimachinery v0.31.1
```

Comment:
Note to reviewers:
The change is generated by the go mod tidy command, the correct versions are pinged under the replace section.